src/blockchain_db/blockchain_db.cpp

```diff
@@ -377,8 +381,22 @@ void BlockchainDB::fixup()
 	set_batch_transactions(true);
 	batch_start();
 
+	static const char* bad_outs[] =
+		{
+			"cbf2af2d517fce9d4d4a34907541c21ce70ea5cfea5bad887022f790c39c93f1",
+			"0a01eaba1c3e73dfe5fc0f70ed9241d06852aadeb5c7905c3fe08731774f5f56",
+			"ba37354aaab679aaf670f76cc225605751a44dab88830016464eb530b4d3be7d"
+		};
+
+	for(const auto &spk : bad_outs)
+	{
+		crypto::public_key pk;
+		epee::string_tools::hex_to_pod(spk, pk);
+		bad_outpks.insert(pk);
+	}
+
 	// block 685498 (13 key images in one transaction)
-	static const char *const premine_key_images[] =
+	static const char* premine_key_images[] =
```

Comment:
why do you removed const from the ptr?

Reply:
Habit mostly - it doesn't _really_ matter if the pointer itself cannot be changed.